GPU/GPUTracking/Base/hip/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ if(ALIGPU_BUILD_TYPE STREQUAL "Standalone")
 endif()
 
 if(HIP_AMDGPUTARGET)
-  target_link_options(${targetName} PUBLIC --amdgpu-target=${HIP_AMDGPUTARGET})
+  target_link_options(${targetName} PUBLIC --amdgpu-target=${HIP_AMDGPUTARGET} -finline-hint-functions)
```

Comment:
Should ` -finline-hint-functions` stay or be removed?

Reply:
this is only a hack for the compiler inside the docker container. I.e. you have to use it in the container, but please do not include it in the PR.